docs/gen_ref_pages.py

```diff
@@ -8,17 +8,22 @@
 import mkdocs_gen_files
 
 
+if not Path('src/easybuild-framework/easybuild').is_dir():
```

Comment:
Maybe checking for `easybuild/framework` is even better?

Reply:
I've made a further change in ee556fc that means that this check is sufficient for the rest of the script to function. If `src/easybuild-framework/easybuild` has nothing in it then we'll generate no API documentation.